packages/cli/src/main/kotlin/elide/tool/cli/cmd/repl/ToolShellCommand.kt

```diff
@@ -1277,13 +1231,14 @@ import org.graalvm.polyglot.Engine as VMEngine
   private fun serveMode(): Boolean = commandSpecifiesServer || executeServe
 
   // Read an executable script, and then execute the script and keep it started as a server.
-  private fun readStartServer(label: String, language: GuestLanguage, ctx: VMContext, source: Source) {
+  private fun readStartServer(label: String, language: GuestLanguage, ctx: PolyglotContext, source: Source) {
     try {
       // enter VM context
       logging.trace("Entered VM for server application (language: ${language.id}). Consuming script from: '$label'")
 
       // initialize the Express intrinsic
-      server.initialize(ctx, phaser.get())
+      // TODO(@darlvd): restore once new server agent implementation lands
```

Comment:
The next PR will reimplement the server agent, using the DSL from there involves a few other changes, and I think it's better to group that with most of the server engine features coming up next.